db/migrate/20170609100310_migrate_public_body_data.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+class MigratePublicBodyData < ActiveRecord::Migration
+  def up
+    PublicBody.find_each do |record|
```

Comment:
I'd like to be able to run this and verify it has worked as expected before immediately dropping the tables (in the next migration). I think we should make this a class in `lib` so that we can add it to a temp rake task that we try before merging & deploying the destructive migrations.

Maybe something along the lines of:

```ruby
# lib/tasks/temp.task
task :migrate_public_body_translations_to_translation_table do
  PublicBodyTranslationMigration.new.migrate_to_translation_table
end

# db/migrate/20170609100310_migrate_public_body_translations.rb
def up
  PublicBodyTranslationMigration.new.migrate_to_translation_table
end

def down
  PublicBodyTranslationMigration.new.revert_to_public_bodies_table
end
```